test/DatasetCustom.js

```diff
@@ -1,21 +1,23 @@
 "use strict";
-const utils = require("./LoginUtils");
+
+var utils = require("./LoginUtils");
 const { TestData } = require("./TestData");
 const { v4: uuidv4 } = require("uuid");
 
-let accessTokenAdminIngestor = null,
-  accessTokenArchiveManager = null,
-  accessTokenUser1 = null,
-  accessTokenUser2 = null,
-
-  pid = null,
-  minPid = null,
-  explicitPid = null;
+var accessTokenAdminIngestor = null;
```

Comment:
I think the use of `var` is discouraged, in favour of `let` https://stackoverflow.com/questions/47465515/why-is-var-not-deprecated